src/autogluon/bench/runbenchmark.py

```diff
@@ -302,25 +303,35 @@ def run(
         aws_config_path = _dump_configs(benchmark_dir=benchmark_dir, configs=aws_configs, file_name="aws_configs.yaml")
 
         if remove_resources:
-            logger.info("Waiting for jobs to complete and then remove the AWS resources created.")
-            logger.info(
-                "You can quit at anytime and run \n"
-                f"`agbench destroy-stack --config_file {aws_config_path}` "
-                "to delete the stack after jobs have run to completion."
-            )
+            wait = True
+        if wait:
+            logger.info("Waiting for jobs to complete. You can quit at anytime and the benchmark will continue to run on the cloud")
+            if remove_resources:
+                logger.info(
+                    "Resources will be deleted after the jobs are finished. You can also call \n"
+                    "`agbench destroy-stack STATIC_RESOURCE_STACK BATCH_STACK CDK_DEPLOY_ACCOUNT CDK_DEPLOY_REGION` "
```

Comment:
Probably a problem with rebase, but can use `agbench destroy-stack --config_file {aws_config_path}` now for simplicity.